homeassistant/components/wemo/entity.py

```diff
@@ -11,17 +11,17 @@
 from homeassistant.helpers.entity import DeviceInfo
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
-from .wemo_device import DeviceWrapper
+from .wemo_device import DeviceCoordinator
 
 _LOGGER = logging.getLogger(__name__)
 
 
 class WemoEntity(CoordinatorEntity):
     """Common methods for Wemo entities."""
 
-    def __init__(self, device: DeviceWrapper) -> None:
+    def __init__(self, device: DeviceCoordinator) -> None:
```

Comment:
Consider renaming `device` to `coordinator`. That's standard.

Reply:
Thanks. Done.